core/templates/span.h

```diff
@@ -103,6 +136,45 @@ constexpr int64_t Span<T>::rfind(const T &p_val, int64_t p_from) const {
 	return -1;
 }
 
+template <typename T>
+constexpr int64_t Span<T>::rfind_seq(const Span<T> &p_span, int64_t p_from) const {
+	const int64_t s = size();
+	const int64_t p_s = p_span.size();
```

Comment:
I think both variables would be better inlined (aka use `size()` instead of `s` everywhere). The `size()` function is guaranteed to be inlined, so it's not a performance concern, and I think `size()` is more readable than `s` :)